client/packages/invoices/src/InboundShipment/api/hooks/utils/useAddFromMasterList.ts

```diff
@@ -33,7 +33,7 @@ export const useAddFromMasterList = () => {
   }: MasterListRowFragment) => {
     getConfirmation({
       onConfirm: () =>
-        mutationState.mutate(
+        mutationState.mutateAsync(
```

Comment:
returning an async function here allows the confirmation modal to show a spinner on clicking `OK`